### PR TITLE
[release/8.0-staging][wasm] WBT Stop taking latest sdk in release/8.0

### PR DESCRIPTION
**Pull request by lewing**
It looks like wasm build tests have started picking up 8.0.2xx sdks after being unpinned.  This pins the WBT sdk to a recent 8.0.1xx sdk which should be new enough for testing and not contain the workload set changes.  We could probably set the channel correctly for 1xx and unpin but that can happen later.

We will need to fix wbt for the workload set changes but that definitely doesn't need to happen before 8.0.0.

Fixes https://github.com/dotnet/runtime/issues/93900

### CODE REVIEW

**Review by radical — APPROVED** [review]

**Comment by lewing** [comment]
@carlossanlop infra only, fixes the wasm failures in release/8.0*

**Comment by carlossanlop** [comment]
Is this blocking release/8.0? If yes, then we should merge, but @mmitche needs to confirm.

Otherwise, can you please target 8.0-staging? Because as of right now, the branch is closed.

**Comment by lewing** [comment]
It is just fixing tests, we can just target staging and ignore the failures if we need to take anything more into release/8.0

**Comment by carlossanlop** [comment]
Okay thanks, please retarget: edit the PR title, switch the branch in the dropdown. Note that _sometimes_ git changes the branch in a bad way and ends up bringing in thinks we don't want, so please double check.

**Comment by lewing** [comment]
I'll retarget once CI finishes then I'd suggest fast merging there

**Comment by lewing** [comment]
This is now all the `release/8.0` to `release/8.0-staging` flow in #93874

**Comment by lewing** [comment]
removed the other commits

**Comment by lewing** [comment]
fast merging to unblock staging

**Pull request merged**
